contributors/devel/sig-architecture/production-readiness.md

```diff
@@ -0,0 +1,51 @@
+# Production Readiness Review Process
```

Comment:
nit: I would move it to sig-architecture/

[there is already api-review process doc there]

Reply:
it's already there?

Reply:
oh, i see

Reply:
I meant not: contributors/devel/sig-architecture, just simply sig-architecture

So basically here:
https://github.com/kubernetes/community/tree/master/sig-architecture